python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/search.py

```diff
@@ -335,3 +342,39 @@ def _parse_facets(self, facets: list[dict[str, Any]]) -> SearchFacets:
             results[field] = options
 
         return SearchFacets(results)
+
+    def _parse_glossary_term(self, entity) -> SearchResult:
+        properties, custom_properties = self._parse_properties(entity)
+        metadata = {"parentNodes": entity["parentNodes"]["nodes"]}
+
+        return SearchResult(
+            id=entity["urn"],
+            result_type=ResultType.GLOSSARY_TERM,
+            matches={},
+            name=properties["name"],
+            description=properties.get("description", ""),
+            metadata=metadata,
+            tags=[],
+            last_updated=None,
+        )
+
+    def get_glossary_terms(self, count: int = 1000) -> SearchResponse:
```

Comment:
I'll wait, just because it's not clear how a larger glossary would be represented on the frontend eg it could be all on one page with a good nav bar/hyperlinks